test/Passes/split-reduction-dim.mlir

```diff
@@ -0,0 +1,332 @@
+// RUN: tpp-opt %s -split-reduction-dim=tile=8 -canonicalize -split-input-file | FileCheck %s
```

Comment:
Pls add a test without canonicalize.

Reply:
I don't think there's anything interesting without it.
Canonicalization just makes IR succinct and the pass has to work anyway with post-cleanup present.

Reply:
In that case, its ok.